client/src/components/commonComponents/Dropdown/index.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './index.css';
+
+const langs = [
+  {
+    id: 1,
+    language: 'Chinese',
+  },
+  {
+    id: 2,
+    language: 'Spanish',
+  },
+  {
+    id: 3,
+    language: 'English',
+  },
+  {
+    id: 4,
+    language: 'Hindi',
+  },
+  {
+    id: 5,
+    language: 'Russian',
+  },
+];
+
+const Dropdown = ({ label, name, languages = langs }) => {
+  return (
+    <div className="drop-down">
+      <label htmlFor="select">
+        {label}
```

Comment:
what do you mean with the label? 
  does it is the HTML tag label ! or do you mean by it the label text ( lblText )